docker-compose.yml

```diff
@@ -32,15 +31,13 @@ services:
     extra_hosts:
       - "host.docker.internal:host-gateway"
     environment:
-      # TODO: dollar signs in the KEY are interpolated, therefore escaped here
-      # with double "$$". Maybe move to .env file to simplify?
-      - SECRET_KEY='django-insecure-w!h85bp^$$e8gm%c23r!0%9i7yzd=6w$$s&ic+6!%306&kj8@k*5'
+      - SECRET_KEY=${SECRET_KEY}
       - DEBUG=True
       - DB_HOST=db
       - DB_PORT=5432
       - DB_NAME=term_db  # see POSTGRES_DB above
       - DB_USER=sadilar  # see POSTGRES_USER above
-      - DB_PASSWORD=sadilar  # see POSTGRES_PASSWORD above
+      - DB_PASSWORD=${DB_PASSWORD}
```

Comment:
You deleted the comment that says "see POSTGRES_PASSWORD above". The point is to keep the two passwords synchronised. I propose you keep the comment, and see the section above that will require the same change.